File: palindromeNumber.py
'''
Given an integer x, return true if x is a palindrome integer.
'''

def isPalindrome(x):
    # Create a list from integer x, reverse it, and compare it to the original.
    x_list = list(str(x))
    list_copy = x_list.copy()
    list_copy.reverse()
    if x_list == list_copy:
        return True
    else:
        return False

File: palindromeNumber.js
/*
Given an integer x, return true if x is a palindrome number.
*/

var isPalindrome = function(x) {
    // Create a list from integer x, reverse it, and compare it to the original.
    let xArray = Array.from(String(x));
    let xCopy = Array.from(xArray);
    xCopy.reverse();
    if (String(xCopy) == String(xArray) {
        return true;
    } else {
        return false;
    }
}

